### PR TITLE
Bug 1918376: ICSP support during Image Registry pullthrough

### PR DESCRIPTION
**Pull request by ricardomaraschini**
This PR adds ICSP support to Image Registry during pullthrough operations.

### CODE REVIEW

File: pkg/dockerregistry/server/pullthroughmanifestservice.go [hunk 1]
```diff
@@ -112,12 +115,13 @@ func (m *pullthroughManifestService) mirrorManifest(ctx context.Context, manifes
 }
 
 func (m *pullthroughManifestService) getRemoteRepositoryClient(ctx context.Context, ref *imageapi.DockerImageReference, dgst digest.Digest, options ...distribution.ManifestServiceOption) (distribution.Repository, error) {
+	dcontext.GetLogger(ctx).Info("(*pullthroughManifestService).getRemoteRepositoryClient")
```
Comment (dmage) [inline pkg/dockerregistry/server/pullthroughmanifestservice.go:118]: That's Debug

Comment (ricardomaraschini) [reply pkg/dockerregistry/server/pullthroughmanifestservice.go]: fixed.

**Comment by dmage** [comment]
Step e2e-aws-upgrade-openshift-e2e-test succeeded

/override ci/prow/e2e-aws-upgrade

**Pull request merged**